src/inputs/manual_input.rs

```diff
@@ -104,6 +104,7 @@ impl ManualInput {
 }
 
 impl InputReader<TdPyAny> for ManualInput {
+    #[tracing::instrument(name = "manual_input", level = "trace", skip_all)]
```

Comment:
This should not be a problem, I just need to add a span where we actually call the python function here. I'll try and let you know

Reply:
Added the child span. In other operator I add the name of the function to the span parameters. Here I'd have to keep around the name of the builder function and add it as a parameter just for that, but since we can only have one input I'm not sure it's worth the trouble.